ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -881,6 +881,11 @@ static PrjFS_Result HandleRecursivelyEnumerateDirectoryRequest(const MessageHead
             
             dirEntry = readdir(directory);
         }
+        
+        if (directory != nullptr)
```

Comment:
We already have a `nullptr` check for this on line 867 and so I don't think we need another one here (`CleanupAndReturn` checks for `nullptr` because it's possible we jump there with a null `directory`).